src/javascript/ContentManager/ContentManager.utils.js

```diff
@@ -59,6 +59,36 @@ function ellipsizeText(text, maxLength) {
     return ellipsize(text, maxLength || 100, {chars: [' ', '&']});
 }
 
+function getZipName(name) {
```

Comment:
I would get rid of the function. It does not bring much value and the naming is quite confusing.
Have the caller use the method below to remove the extension if any, and then append '.zip'